### PR TITLE
Update travis with node versions and npm dependencies

### PR DESCRIPTION
**Pull request by Hkly**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,13 +4,11 @@
   "description": "Limit the number of promises that can be in flight simultaneously",
   "main": "index.js",
   "dependencies": {
-    "chai": "^2.2.0",
-    "chai-as-promised": "^4.3.0",
     "q": "^1.2.0"
```
Comment (LionOps) [inline package.json]: Why not update this?

Comment (Hkly) [inline package.json]: Weird. I was depending on `npm outdated` but I guess that's not 100% accurate, or it doesn't do what I think it should do.. heh updating.

File: .travis.yml [hunk 2]
```diff
@@ -3,6 +3,8 @@ language: node_js
 node_js:
   - '0.10'
   - '0.12'
+  - '4'
+  - '5'
```
Comment (LionOps) [inline .travis.yml]: `stable`

**Comment by LionOps** [comment]
:+1: 

I'd rebase so that there are only two commits.

**Pull request merged**